src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -225,57 +227,61 @@ $.BootstrapTable = class extends $.BootstrapTable {
             filterValue = Utils.normalizeAccent(filterValue)
           }
 
-          if (filterValue === '') {
-            tmpItemIsExpected = true
-          } else {
-            // Fix #142: search use formatted data
-            if (thisColumn) {
-              if (thisColumn.searchFormatter || thisColumn._forceFormatter) {
-                value = $.fn.bootstrapTable.utils.calculateObjectValue(
-                  that.header,
-                  that.header.formatters[$.inArray(key, that.header.fields)],
-                  [value, item, i],
-                  value
-                )
-              }
+          let filterValues = [filterValue]
+
+          if (
+            this.options.filterControlMultipleSearch
+          ) {
+            filterValues = filterValue.split(this.options.filterControlMultipleSearchDelimiter)
```

Comment:
I think it should trim spaces before and after the comma because if I put this Item 1, Item 2 it doesn't work